### PR TITLE
Integrate Sentry

### PR DESCRIPTION
**Pull request by alex-tiutiunnyk**

### CODE REVIEW

File: slidez-core/build.gradle [hunk 1]
```diff
@@ -36,6 +36,7 @@ dependencies {
 	implementation 'org.springdoc:springdoc-openapi-data-rest:1.5.9'
 	implementation 'org.springdoc:springdoc-openapi-security:1.5.9'
 	implementation 'org.flywaydb:flyway-core'
+    implementation 'io.sentry:sentry-spring:5.0.1'
```
Comment (Anatolii-Ivanov) [inline slidez-core/build.gradle:39]: Check tabulation please

File: slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java [hunk 2]
```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import io.sentry.spring.EnableSentry;
 
 @SpringBootApplication
+@EnableSentry(dsn = "https://b579813c296046399a14368064320494@o938867.ingest.sentry.io/5888744")
```
Comment (Anatolii-Ivanov) [inline slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java:8]: Isn't it possible to get this value from config? As I seems to be unique for each user

File: slidez-fe-core/src/index.tsx [hunk 3]
```diff
@@ -5,6 +5,14 @@ import App from './App'
 import reportWebVitals from './reportWebVitals'
 import { store } from './store'
 import { Provider } from 'react-redux'
+import * as Sentry from '@sentry/react'
+import { Integrations } from '@sentry/tracing'
+
+Sentry.init({
+    dsn: 'https://b579813c296046399a14368064320494@o938867.ingest.sentry.io/5888744',
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/index.tsx:12]: Move to config?

Comment (alex-tiutiunnyk) [reply slidez-fe-core/src/index.tsx]: I moved it, please check

**Review by Anatolii-Ivanov — APPROVED** [review]

File: slidez-core/src/main/resources/application.properties [hunk 4]
```diff
@@ -25,3 +24,9 @@ spring.flyway.enabled=true
 #-------------------------
 springdoc.api-docs.path=/api-docs
 springdoc.swagger-ui.path=/swagger-ui
+#-------------------------
+# Sentry
+#-------------------------
+sentry.dsn=https://b579813c296046399a14368064320494@o938867.ingest.sentry.io/5888744
+sentry.in-app-includes=com.binarystudio.academy.slidez
+
```
Comment (Anatolii-Ivanov) [inline slidez-core/src/main/resources/application.properties:32]: Move it to staging properties, as we don't want to get exception message every time somebody debug something :)

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: slidez-core/src/main/resources/application.properties (hunk 4 above)
Comment (alex-tiutiunnyk) [reply slidez-core/src/main/resources/application.properties:32]: Yeah, it was there, but after refactoring it didn't work this way. So i found the possible variant with sentry.properties file and it worked, but in call with Kirill he told me to put sentry dsn key only for application.properties. But if not, let me know this and i will change it)

Comment (Anatolii-Ivanov) [reply slidez-core/src/main/resources/application.properties]: Yes, move them to application.staging.properties and remove from application.properties.example

Comment (alex-tiutiunnyk) [reply slidez-core/src/main/resources/application.properties]: I changed it, check please

File: slidez-core/src/main/resources/application.properties [hunk 5]
```diff
@@ -18,15 +16,8 @@ spring.jpa.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.flyway.enabled=true
-
 #-------------------------
 # OpenAPI links
 #-------------------------
 springdoc.api-docs.path=/api-docs
 springdoc.swagger-ui.path=/swagger-ui
```
Comment (Anatolii-Ivanov) [inline slidez-core/src/main/resources/application.properties]: I think we don't have this file on develop

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by annaprusakova — APPROVED** [review]

**Pull request merged**